src/libraries/System.Reflection.Emit.ILGeneration/tests/ILGenerator/Emit5Tests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public sealed class ILGeneratorEmit5
```

Comment:
Not the most descriptive class name, something like "EmitWithMods" might be better, but I guess it is consistent with the existing ones.

Reply:
Actually i agree, I'll change this